config/sync/core.entity_view_display.config_pages.stanford_local_footer.default.yml

```diff
@@ -170,12 +183,16 @@ content:
     weight: 3
     label: hidden
     settings:
-      trim_length: 80
+      trim_length: null
       url_only: false
       url_plain: false
-      rel: ''
-      target: ''
-    third_party_settings: {  }
+      rel: '0'
+      target: '0'
+    third_party_settings:
+      field_formatter_class:
```

Comment:
I really hate this. it adds an unnecessary dependency on the module.

Reply:
Ditto. I suspect we could trim it out but that would be a whole whack of manual labor each time.